src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: $width-of-items;
+    font-weight: 500;
+    font-size: $font-size-12;
+    line-height: 18px;
+    color: $color-black;
+
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    width: $width-of-items;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-grey;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    width: $width-of-items;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews-label {
+    font-size: 10px;
+  }
+
+  &__price {
+    width: $width-of-items;
+    height: 18px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: $font-size-12;
+    color: $color-grey;
+  }
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $color-black;
+  }
+
+  &__button {
+    text-decoration: none;
+
+    width: $width-of-items;
+    height: 40px;
+    border-radius: 5px;
+    background-color: rgb(0, 172, 220);
+    color: white;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+    line-height: 40px;
+
+    &:hover {
+      box-sizing: border-box;
+      background-color: #fff;
+      color: rgb(0, 172, 220);
+      border: 1px solid rgb(0, 172, 220);
+    }
```

Comment:
The `&__button` class seems to be doing too much. Consider breaking it down into smaller, reusable classes. For example, you could create a separate class for the button's base styling (like background color, text color, font size, etc.) and another class for its hover state. This is a code quality issue.